app/src/main/java/com/gh4a/BaseActivity.java

```diff
@@ -275,6 +277,11 @@ protected void toggleRightSideDrawer() {
     }
 
     protected void configureLeftDrawerHeader(View header) {
+        ViewCompat.setOnApplyWindowInsetsListener(header, (view, insets) -> {
+            int statusBarHeight = insets.getInsets(WindowInsetsCompat.Type.systemBars()).top;
+            view.setPadding(0, statusBarHeight, 0, 0);
```

Comment:
Won't this kill the left and bottom padding set in XML?

Reply:
Yes, but the root layout has no padding -> https://github.com/slapperwan/gh4a/blob/6080d0e6ada126ce70654b7fdd3f319bbff83e9a/app/src/main/res/layout/drawer_header_left.xml#L8